docs/hubs-discord-bot.md

```diff
@@ -4,44 +4,68 @@ title: Discord Bot
 ---
 
 ## About
-The [Hubs Discord Bot](https://github.com/Hubs-Foundation/hubs-discord-bot) makes it easy to connect Hubs rooms to your Discord chat server. When a Hubs room is associated with a Discord channel, users will be assigned abilities in the Hubs room based on their Discord roles. For example, Discord owners and moderators will be able to change settings on a Hubs room and be able to moderate users in the room.
+
+The [Hubs Discord Bot](https://github.com/Hubs-Foundation/hubs-discord-bot) makes it easy to connect Hubs rooms to your Discord chat server. When a Hubs room is created by the bot from within a Discord channel, users will be assigned abilities in the Hubs room based on their Discord roles. For example, Discord owners and moderators will be able to change settings on a Hubs room and be able to moderate users in the room.  Chat messages/images will also be bridged both ways between Discord and the Hubs room.  If a Room URL not created by the bot is added to a Discord channel topic, the chat will be bridged, but it will use the standard Hubs permissions.
```

Comment:
This is fine as-is, but I would lead with a sentence containing  "Allows you to authenticate users"

Reply:
How about `When a Hubs room is created by the bot from within a Discord channel, users will be required to authenticate via Discord and then assigned abilities in the Hubs room based on their Discord roles`?  The Discord bot does two separate things (bridging the chat, and authenticating users) that, as best as I can tell, are mutually exclusive from each other, so there isn't a single primary function.